shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -17,7 +17,7 @@
     {{/if}}
   {{else}}
     {{#if isCurrentRoute 'shared'}}
-      <a class="highlight">Save this link ▾</a>
+      <a class="highlight">Sign in ▾</a>
```

Comment:
Do we still want to highlight this? Maybe we should remove the conditional altogether?